.github/scripts/check_icon_pr.py

```diff
@@ -105,12 +123,37 @@ def check_devicon_object(icon: dict):
 
     try:
         if type(icon["aliases"]) != list:
-            err_msgs.append("- 'aliases' must be an array.")
+            err_msgs.append("- \"aliases\" must be an array of objects.")
     except KeyError:
         err_msgs.append("- missing key: 'aliases'.")
-    
+
+    try:
```

Comment:
Shouldn't we check if the `alias` is the same as an existent icon version? If so, an error must be shown, I guess.

Reply:
Yes, but I think @lunatic-fox fixes this in another PR. I can't remember which one though